### ISSUE TITLE
profiling slow methods

### ISSUE THREAD

**Issue by renepickhardt** [renepickhardt/generalized-language-modeling-toolkit#9]
with the help of a profiler we can watch out for performance bottlenecks and then increase the performance of certain methods (this will be tricky due to disc / io)

**Comment by lschmelzeisen** [comment]
There has been quite some work done on getting the programm fast. I guess we should delay any further optimizations after the release?

**Comment by renepickhardt** [comment]
agree. optimizing is also something that partly relates to your bachlor
thesis

On Tue, Jan 6, 2015 at 3:39 PM, Lukas Schmelzeisen <notifications@github.com

> wrote:
> 
> There has been quite some work done on getting the programm fast. I guess
> we should delay any further optimizations after the release?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/renepickhardt/generalized-language-modeling-toolkit/issues/9#issuecomment-68872683
> .

## 

## 

www.rene-pickhardt.de
http://www.beijing-china-blog.com/

Skype: rene.pickhardt

mobile: +49 (0)157 / 3730 2422    office: +49 (0) 261 / 287 2765    fax:
+49 (0) 267 / 287 100 2765

china mobile: +86 186 2129 5033
